apps/postybirb-ui/project.json

```diff
@@ -76,6 +76,12 @@
           "buildTarget": "postybirb-ui:build:production"
         }
       }
+    },
+    "typecheck": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "tsc -b {projectRoot}/tsconfig.json --incremental --pretty"
+      }
```

Comment:
There is no offical executor, atleast in this nx version. Also i added typecheck only to the apps, not libs. Not sure if libs need it anyway